index.d.ts

```diff
@@ -69,18 +75,18 @@ declare namespace Eris {
   type ActionRowComponents = Button | SelectMenu;
   type Button = InteractionButton | URLButton;
   type Component = ActionRow | ActionRowComponents;
-  type ImageFormat = "jpg" | "jpeg" | "png" | "gif" | "webp";
+  type ImageFormat = Constants["ImageFormats"][number];
   type MessageContent = string | AdvancedMessageContent;
-  type MFALevel = 0 | 1;
+  type MFALevel = (Constants["MFALevel"])[keyof Constants["MFALevel"]];
```

Comment:
```suggestion
  type MFALevel = Constants["MFALevel"][keyof Constants["MFALevel"]];
```
You guessed it, here included